[2024-02-08 12:19:42,853: INFO: main: Welcome to our cutom log]
[2024-02-08 12:21:07,471: INFO: main: Welcome to our cutom log]
[2024-02-08 12:22:08,200: INFO: main: Welcome to our cutom log]
[2024-02-24 15:22:19,491: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 15:22:19,506: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 15:22:19,509: INFO: common: created directory at: artifacts]
[2024-02-24 15:22:19,513: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 15:22:19,516: INFO: 2161667167: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 15:25:23,599: INFO: 2161667167: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 15:49:40,173: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-24 15:49:40,177: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 15:49:40,178: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 15:49:40,180: INFO: common: created directory at: artifacts]
[2024-02-24 15:49:40,181: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 15:49:40,182: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 15:52:32,942: INFO: data_ingestion: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 15:52:54,680: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2024-02-24 16:47:37,487: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:47:37,498: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:47:37,501: INFO: common: created directory at: artifacts]
[2024-02-24 16:47:37,503: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:49:23,175: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:49:23,179: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:49:23,180: INFO: common: created directory at: artifacts]
[2024-02-24 16:49:23,181: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:52:41,306: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:52:41,311: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:52:41,313: INFO: common: created directory at: artifacts]
[2024-02-24 16:52:41,315: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:54:50,245: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:54:50,248: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:54:50,250: INFO: common: created directory at: artifacts]
[2024-02-24 16:54:50,251: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:58:00,202: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:58:00,207: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:58:00,208: INFO: common: created directory at: artifacts]
[2024-02-24 16:58:00,210: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:58:09,934: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:58:09,937: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:58:09,939: INFO: common: created directory at: artifacts]
[2024-02-24 16:58:09,940: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 16:59:50,291: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 16:59:50,295: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 16:59:50,297: INFO: common: created directory at: artifacts]
[2024-02-24 16:59:50,299: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 17:00:05,462: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-24 17:18:01,855: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-24 17:18:01,857: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 17:18:01,859: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 17:18:01,860: INFO: common: created directory at: artifacts]
[2024-02-24 17:18:01,861: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 17:18:01,861: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 17:21:04,507: INFO: data_ingestion: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 17:21:14,456: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2024-02-24 17:21:14,457: INFO: main: *************************]
[2024-02-24 17:21:14,457: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-24 17:21:14,459: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 17:21:14,460: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 17:21:14,462: INFO: common: created directory at: artifacts]
[2024-02-24 17:21:14,462: ERROR: main: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config']
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "c:\projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 14, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config'
[2024-02-24 17:28:55,036: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-24 17:28:55,038: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 17:28:55,039: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 17:28:55,040: INFO: common: created directory at: artifacts]
[2024-02-24 17:28:55,041: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 17:28:55,041: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 17:31:59,302: INFO: data_ingestion: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 17:32:08,871: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2024-02-24 17:32:08,871: INFO: main: *************************]
[2024-02-24 17:32:08,871: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-24 17:32:08,873: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 17:32:08,874: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 17:32:08,875: INFO: common: created directory at: artifacts]
[2024-02-24 17:32:08,875: ERROR: main: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config']
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "c:\projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 15, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config'
[2024-02-24 20:05:59,635: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-24 20:05:59,638: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 20:05:59,642: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 20:05:59,642: INFO: common: created directory at: artifacts]
[2024-02-24 20:05:59,643: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 20:05:59,644: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 20:09:04,468: INFO: data_ingestion: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 20:09:17,263: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2024-02-24 20:09:17,264: INFO: main: *************************]
[2024-02-24 20:09:17,264: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-24 20:09:17,267: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 20:09:17,269: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 20:09:17,270: INFO: common: created directory at: artifacts]
[2024-02-24 20:09:17,270: ERROR: main: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config']
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "c:\projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 15, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config'
[2024-02-24 20:14:12,258: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-24 20:14:12,261: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 20:14:12,263: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 20:14:12,264: INFO: common: created directory at: artifacts]
[2024-02-24 20:14:12,265: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-24 20:14:12,265: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 20:17:35,574: INFO: data_ingestion: Download data from https://drive.google.com/file/d/1NNngfD6Tp2bl-iMN_sIAWpSiSdFfwvuG/view into file artifacts/data_ingestion/data.zip]
[2024-02-24 20:17:52,331: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2024-02-24 20:17:52,331: INFO: main: *************************]
[2024-02-24 20:17:52,332: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-24 20:17:52,334: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-24 20:17:52,336: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-24 20:17:52,336: INFO: common: created directory at: artifacts]
[2024-02-24 20:17:52,337: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-24 20:17:53,098: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-24 20:17:53,438: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
